src/frontend/src/components/cardComponent/index.tsx

```diff
@@ -177,12 +177,12 @@ export default function CollectionCardComponent({
                   )}
                   Playground
                 </Button>
-              )}
+              )} */}
             </div>
           </div>
         </CardFooter>
       </Card>
-      {openPlayground && (
+      {/* {openPlayground && (
```

Comment:
can we remove these comments?

Reply:
we probably will bring this feature back in a near future, that's why I want to keep it commented